app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with model: [ curiosity, comment ], html: {
+  data: { turbo: false } } do |form| %>
```

Comment:
Is there a reason why we're disabling turbo here?

Reply:
oh yes, It's because haml doesn't support turbo yet, but I´ll change it later by .erb  extension to avoid this heh :D